tensornetwork/contractors/opt_einsum_paths/path_contractors_test.py

```diff
@@ -92,3 +92,21 @@ def __call__(self, inputs, output, size_dict, memory_limit=None):
   optimizer = PathOptimizer()
   final_node = path_contractors.custom(net, optimizer).get_final_node()
   np.testing.assert_allclose(final_node.tensor, np.ones(5) * 2.0)
+
+
+def test_ignore_edge_order(backend, path_algorithm):
+  net = tensornetwork.TensorNetwork(backend=backend)
+  a = net.add_node(np.ones((1, 1, 1)))
+  b = net.add_node(np.ones((1, 1, 1, 2, 3)))
+
+  a[0] ^ b[0]
+  a[1] ^ b[1]
+  a[2] ^ b[2]
+
+  e0 = b[3]
+  e1 = b[4]
+
+  final_node = path_algorithm(net,
+                              [e1, e0],
+                              ignore_edge_order=True).get_final_node()
```

Comment:
Seems kinda pointless to test `ignore_edge_order=True` when an edge order is given anyway. :)

Reply:
I see what you mean, but it also ensures it's effective despite being passed an order, which I think is a valid test case.

Reply:
Can you add another similar test where you don't include the list of edges?

Reply:
Sure thing, I'll add to path_contractors_node_test as detailed below.